### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by smira**
| Package | Update | Change |
|---|---|---|
| git://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git | minor | `20230515` -> `20230625` | 
| git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git | patch | `6.1.35` -> `6.1.37` | 
| git://git.kernel.org/pub/scm/utils/util-linux/util-linux.git | patch | `2.39` -> `2.39.1` | 
| https://github.com/ipxe/ipxe.git | digest | `2689a6e` -> `e17568a` |

### CODE REVIEW

**Review by DmitriyMV — APPROVED** [review]

File: kernel/build/config-amd64 [hunk 1]
```diff
@@ -1013,6 +1013,7 @@ CONFIG_SECRETMEM=y
 CONFIG_LRU_GEN=y
 # CONFIG_LRU_GEN_ENABLED is not set
 # CONFIG_LRU_GEN_STATS is not set
+CONFIG_LOCK_MM_AND_FIND_VMA=y
```
Comment (frezbo) [inline kernel/build/config-amd64:1016]: not cleaned up from previous PR?

**Review by frezbo — APPROVED** [review]

File: kernel/build/config-amd64 (hunk 1 above)
Comment (smira) [reply kernel/build/config-amd64:1016]: I think it's new Linux

Comment (smira) [reply kernel/build/config-amd64:1016]: I ran `olddefconfig` on the previous PR

**Pull request merged**